src/cfml/system/modules_app/server-commands/commands/server/prune.cfc

```diff
@@ -0,0 +1,59 @@
+/**
+ * Forget old servers which haven't been started in a given period of time. time will be calculated in days.
+ * .
+ * {code:bash}
+ * server prune days=30
+ * server prune days=30 --list
```

Comment:
Remove `--list` from examples since it's no longer part of the command

Reply:
done